pkgs/tools/misc/zalgo/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "zalgo-unstable-2020-08-26";
```

Using `name` is deprecated.
```suggestion
  pname = "zalgo-unstable";
  version = "2020-08-26";
```